routes/campgrounds.js

```diff
@@ -27,35 +27,55 @@ cloudinary.config({
 
 // INDEX - show all campgrounds in the web
 router.get('/', (req, res) => {
+  const perPage = 8;
+  const pageQuery = parseInt(req.query.page);
+  const pageNumber = pageQuery ? pageQuery : 1;
```

Comment:
Could get rid of `pageQuery` variable by just doing this instead:
`const pageNumber = parseInt(req.query.page) || 1;`